dhis-2/dhis-api/src/main/java/org/hisp/dhis/option/OptionSet.java

```diff
@@ -122,6 +123,14 @@ public void removeAllOptions()
         options.clear();
     }
 
+    public void removeOption( Option option )
```

Comment:
This is a little bit worrying as we might be hiding underlying issues. Do we know why the options list is null here?

Reply:
@larshelge  I talked with Pamod yesterday. They can't reproduce the issue after restarting the sever. All OptionSets are now working normally on their instance. 
My theory is that after executing the clean_up_option_sortOrder script, it reset all the OptionSet.options references in the cache. But I couldn't replicate this behaviour on my local. So not really sure what happened on their instance.
I will update the clean up script document to add a final step for restarting server.